config/jobs/kubernetes-sigs/kjob/kjob-periodics-release-0.1.yaml

```diff
@@ -117,8 +207,53 @@ periodics:
             privileged: true
           resources:
             requests:
-              cpu: "7"
-              memory: "10Gi"
+              cpu: "2"
+              memory: "4Gi"
```

Comment:
don't mix this change, let's do a preparatory PR

Reply:
Done

Reply:
https://github.com/kubernetes/test-infra/pull/34588